.projenrc.js

```diff
@@ -10,9 +10,8 @@ const project = new awscdk.AwsCdkConstructLibrary({
   description: 'The CDK Construct Library that helps you build ECS services using simple extensions',
   name: 'cdklabs/cdk-ecs-service-extensions',
   repositoryUrl: 'https://github.com/cdklabs/cdk-ecs-service-extensions.git',
-  stability: 'experimental',
+  stability: 'stable',
   majorVersion: 2,
```

Comment:
if 2.1 is the version we seek, have you confirmed this? We can enable `dryRun` to test it out before if we are worried.

Reply:
Yeah, let's do that to check. I don't want to accidentally re-release 2.0.0 or have a patch release go out.

Reply:
Do you mean `publishDryRun` in the `.projenrc.js` settings?

Reply:
i believe so :).